### DISCUSSION TITLE
Integrating Azure B2C for Azure DICOM Service: Can B2C replace Entra ID?

### DISCUSSION THREAD

**Discussion by daniel-evodicom** [microsoft/dicom-server#3255] [Q&A]
I am building an RIS, and my authentication method is through Azure B2C. Now I want to implement Azure DICOM Service, and I understand that it requires authentication through Entra ID. Can I use B2C instead of Entra ID to use a single token instead of two? If it is possible, what is required, or what changes with respect to the flow described here?

**Comment by arunmk-ms** [comment]
Hi,

Can you please let us know how you are deploying Dicom service? Currently our [managed service](https://github.com/microsoft/dicom-server?tab=readme-ov-file#managed-service) supports only Entra Id authentication.
